src/commands/firestore-databases-update.ts

```diff
@@ -0,0 +1,81 @@
+import { Command } from "../command";
+import * as clc from "colorette";
+import * as fsi from "../firestore/api";
+import * as types from "../firestore/api-types";
+import { logger } from "../logger";
+import { requirePermissions } from "../requirePermissions";
+import { Emulators } from "../emulator/types";
+import { warnEmulatorNotSupported } from "../emulator/commandUtils";
+import { FirestoreOptions } from "../firestore/options";
+
+export const command = new Command("firestore:databases:update")
+  .description(
+    "Update a database in your Firebase project. Must specify at least one property to update."
+  )
+  .option(
+    "--pretty",
+    "Pretty print. When not specified the databases are printed in the " +
+      "JSON specification format."
+  )
+  .option("--database <databaseId>", "Name of database to be updated. (required)")
+  .option(
```

Comment:
Nit: If I'm reading the code correctly, type can be updated, but its not obvious from this help text. Consider clarifying to something like ie `Type to update the database to` - with the current wording, it sound like i'm supposed to provide the current type/